python/pyarrow/tests/test_parquet.py

```diff
@@ -366,6 +366,20 @@ def test_multithreaded_read():
 
 
 @parquet
+def test_min_chunksize():
+    data = pd.DataFrame([np.arange(4)], columns=['A', 'B', 'C', 'D'])
+    table = pa.Table.from_pandas(data.reset_index())
+
+    buf = io.BytesIO()
+    pq.write_table(table, buf, chunk_size=0)
```

Comment:
Neither value makes sense logically, but -1 seems like a more obvious indicator that we're saying "do the default".